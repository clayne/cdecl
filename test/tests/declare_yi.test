EXPECTED_EXIT=0
cdecl <<END
declare final as int
END
