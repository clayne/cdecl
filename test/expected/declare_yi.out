declare final as int
        ^
9: warning: "final" is a keyword in C++11
int final;
